commit 918cf9bb38de66fe30d58f3c7c680bbdbae463d2 (refs/changes/98/598/1)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2016-10-04T23:01:25-07:00 (3 years ago)
    
    PUBAPI-1218 ListMachines docs and behaviour of 'tombstone' option do not agree

diff --git a/bin/sdc-listmachines b/bin/sdc-listmachines
index f41fa93..4627283 100755
--- a/bin/sdc-listmachines
+++ b/bin/sdc-listmachines
@@ -27,7 +27,7 @@ var Options = {
     'state': String,
     'tag': [String, Array],
     'type': String,
-    'tombstone': Number,
+    'tombstone': Boolean,
     'url': url,
     'version': Boolean,
     'verbose': Boolean,
diff --git a/lib/cli_common.js b/lib/cli_common.js
index 1fad45e..5398e84 100644
--- a/lib/cli_common.js
+++ b/lib/cli_common.js
@@ -62,7 +62,7 @@ var MasterOptions = {
         'the state of the instance (i.e. running, stopped ...) (default: all)',
     type: 'filter by type (default: all)',
     tombstone:
-        'include instances destroyed in the last N minutes (default: none)',
+        'include destroyed and failed instances available in instance history (default: false)',
     image: 'the machine image id (UUID)',
     // snapshot
     snapshot: 'snapshot name',
